diff --git a/dist/es2015/component.js b/dist/es2015/component.js
index 4a7ee5b055a3c1e26445977eea909e369aee857b..d8a8b7efd5be4c3c384d3661965f7ec09daee03d 100644
--- a/dist/es2015/component.js
+++ b/dist/es2015/component.js
@@ -49,5 +49,5 @@ export var RemoveScrollBar = function (_a) {
      due to singleton nature of <Style
      */
     var gap = React.useMemo(function () { return getGapWidth(gapMode); }, [gapMode]);
-    return React.createElement(Style, { styles: getStyles(gap, !noRelative, gapMode, !noImportant ? '!important' : '') });
+    return React.createElement(Style, { styles: getStyles(gap, !noRelative, gapMode, !noImportant ? '' : '') });
 };
diff --git a/dist/es2019/component.js b/dist/es2019/component.js
index 96f6bbb506e7c26e46def0dc1876a34d349ea221..14e2ab26561ca97623500e3ef8f2887d2d757f33 100644
--- a/dist/es2019/component.js
+++ b/dist/es2019/component.js
@@ -81,5 +81,5 @@ export const RemoveScrollBar = ({ noRelative, noImportant, gapMode = 'margin' })
      due to singleton nature of <Style
      */
     const gap = React.useMemo(() => getGapWidth(gapMode), [gapMode]);
-    return React.createElement(Style, { styles: getStyles(gap, !noRelative, gapMode, !noImportant ? '!important' : '') });
+    return React.createElement(Style, { styles: getStyles(gap, !noRelative, gapMode, !noImportant ? '' : '') });
 };
diff --git a/dist/es5/component.js b/dist/es5/component.js
index c29d6d8fe783a81bf1468511070f47db5c2fbacc..41a97801ee3942889e333f580282520e69622bcd 100644
--- a/dist/es5/component.js
+++ b/dist/es5/component.js
@@ -54,6 +54,6 @@ var RemoveScrollBar = function (_a) {
      due to singleton nature of <Style
      */
     var gap = React.useMemo(function () { return (0, utils_1.getGapWidth)(gapMode); }, [gapMode]);
-    return React.createElement(Style, { styles: getStyles(gap, !noRelative, gapMode, !noImportant ? '!important' : '') });
+    return React.createElement(Style, { styles: getStyles(gap, !noRelative, gapMode, !noImportant ? '' : '') });
 };
 exports.RemoveScrollBar = RemoveScrollBar;
